podcasts/Onboarding/Plus/UpgradeLandingView.swift

```diff
@@ -159,7 +158,7 @@ extension UpgradeTier {
     }
 
     static var patron: UpgradeTier {
-        UpgradeTier(tier: .patron, iconName: "patron-heart", title: "Patron", yearlyIdentifier: "com.pocketcasts.patron.yearly", monthlyIdentifier: "com.pocketcasts.patron.monthly", description: L10n.patronDescription, buttonLabel: L10n.patronSubscribeTo, buttonColor: Color(hex: "6046F5"), buttonForegroundColor: .white, features: [
+        UpgradeTier(tier: .patron, iconName: "patron-heart", title: "Patron", plan: .patron, description: L10n.patronDescription, buttonLabel: L10n.patronSubscribeTo, buttonColor: Color(hex: "6046F5"), buttonForegroundColor: .white, features: [
```

Comment:
Should we add Patron to the localized strings?

Reply:
Hmm, we can check. But I don't think we want to translate "Patron".

Reply:
Same as [above](https://github.com/Automattic/pocket-casts-ios/pull/822#discussion_r1165837136).